yarn-project/pxe/src/pxe_service/pxe_service.ts

```diff
@@ -892,11 +892,6 @@ export class PXEService implements PXE {
         if (!visibleEvent.eventTypeId.equals(eventMetadata.eventSelector)) {
           return undefined;
         }
-        if (visibleEvent.event.items.length !== eventMetadata.fieldNames.length) {
```

Comment:
This was incorrect because on the left we have an event serialized to fields and on the right we have the abi members which can occupy more than 1 field (as is the case with U128 where we occupy 2 fields for low and high limbs).